### ISSUE TITLE
Contrast is bad when showing errors in dark mode

### ISSUE THREAD

**Issue by davidfowl** [dotnet/aspire#1627]
<img width="1030" alt="image" src="https://github.com/dotnet/aspire/assets/95136/4a23af6b-bd19-4288-85ec-e2f750264821">

black on bright red of rough on the eyes. Can we use white text instead or make the read softer?
